### ISSUE TITLE
Getting team list prints <nil> values to command line when account is unapproved

### ISSUE THREAD

**Issue by mroseman95** [manifoldco/torus-cli#267]
Reproduce:

1. Create an account and org
2. Invite another email to org
3. Create account with second email
4. Accept the invite, but don't approve it
5. view team members with `torus-cli teams members member`
6. get output 
```
Org not found.
<nil>
<nil>
<nil>
```

System state:
```
Timestamp            Sat Jun 17 22:51:23 UTC 2017
Debug                false
Session Type         user
User ID              040wveku9txzfma6n4wb703h65k92
Full Name <Email>    Matthew Roseman <mrr77@case.edu>
Identity             test2
                      
CLI                  alpha
Daemon               alpha
Registry             0.20.0
Registry URI         https://registry.arigato.sh
                      
Org                  stohio
Project              manifold-quickstart
Environment          dev-test2
Service              default
Instance             1
```

**Comment by ianlivingstone** [comment]
Sweet,thanks for the bug report!

**Comment by mroseman95** [comment]
This also happens for `torus-cli teams list`. I get output
```
Org not found.
<nil>
<nil>
Error fetching teams list.
```
I'd imagine these aren't the only two commands that have similar outputs for unapproved users, but I don't know the commands well enough to test them all.

**Comment by ianlivingstone** [comment]
Fixed by #345
